content/project/governance-meeting/index.adoc

```diff
@@ -15,7 +15,7 @@ If a decision cannot be made at the governance meeting, the project's link:/proj
 
 === How to join the meetings?
 
-The governance meeting commonly happens every two weeks on Wednesdays, see the link:/events[event calendar] for exact times and subscription.
+The governance meeting commonly happens every two weeks on Monday, see the link:/events[event calendar] for exact times and subscription.
```

Comment:
Use same plural as before.  No idea if one is preferred over the other among those who understand English grammar
```suggestion
The governance meeting commonly happens every two weeks on Mondays, see the link:/events[event calendar] for exact times and subscription.
```